src/System.Private.CoreLib/shared/System/SpanHelpers.Byte.cs

```diff
@@ -111,9 +111,20 @@ public static unsafe bool Contains(ref byte searchSpace, byte value, int length)
             IntPtr offset = (IntPtr)0; // Use IntPtr for arithmetic to avoid unnecessary 64->32->64 truncations
             IntPtr lengthToExamine = (IntPtr)length;
 
-            if (Vector.IsHardwareAccelerated && length >= Vector<byte>.Count * 2)
+            if (Avx2.IsSupported || Sse2.IsSupported)
```

Comment:
```suggestion
            if (Sse2.IsSupported)
```
`Avx2.IsSupported == true` implies `Sse2.IsSupported == true`, so this can be simpliefied, as the Avx2-codepath isn't separate in this piece of code.